patches/vite-plugin-solid+2.11.0.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/node_modules/vite-plugin-solid/dist/esm/index.mjs b/node_modules/vite-plugin-solid/dist/esm/index.mjs
+index e065926..ee9e2bd 100644
+--- a/node_modules/vite-plugin-solid/dist/esm/index.mjs
++++ b/node_modules/vite-plugin-solid/dist/esm/index.mjs
```

Comment:
Is the Solid team aware of the issue and PR? Maybe reproducing the problem could help expedite things?

Reply:
Yeah, Ryan told me it looks good to him in a private channel where I shared this, but afaik that plugin is a bit stuck in terms of landing PRs. I will try to ping some folks to hopefully get it merged though.